File: cfgs/java_cfgs_foo/COUNT_NUMBERS_THAT_DONT_CONTAIN_3.java
Nodes
1;special;;;
2;conditional;foo_1 < 3;14;
3;statement;return foo_1;14;
4;exit;;;
5;conditional;(foo_1 >= 3) && (foo_2 < 10);15;
6;statement;return foo_1 - 1;15;
7;statement;int po = 1;16;
8;conditional;(foo_1 / foo_2) > 9;17;
9;statement;po = foo_1 * 10;17;
10;statement;int msd = foo_1 / foo_2;18;
11;conditional;foo_1 != 3;19;
12;statement;return ((fun_1(foo_1) * fun_2(foo_2 - 1)) + fun_3(foo_3)) + fun_4(foo_4 % foo_5);19;
13;statement;return fun_1((foo_1 * foo_2) - 1);20;
Edges
0;2;
1;3;5;
2;4;
3;
4;6;7;
5;4;
6;8;
7;10;9;
8;8;
9;11;
10;12;13;
11;4;
12;4;

File: cfgs/pythoncfgs/COUNT_NUMBERS_THAT_DONT_CONTAIN_3.py
Nodes
1;conditional;foo_1 < 3;8;
2;conditional;foo_1 >= 3 and foo_2 < 10;10;
3;statement;foo_1 = 1;12;
4;conditional;foo_1 / foo_2 > 9;13;
5;statement;foo_1 = foo_2 / foo_3;15;
6;conditional;foo_1 != 3;16;
7;statement;return fun_1(foo_1 * foo_2 - 1);19;
8;statement;return fun_1(foo_1) * fun_2(foo_2 - 1) + fun_3(foo_3) + fun_4(foo_4 % foo_5);17;
9;statement;foo_1 = foo_2 * 10;14;
10;statement;return foo_1 - 1;11;
11;statement;return foo_1;9;
12;exit;;
Edges
0;2;11;
1;3;10;
2;4;
3;5;9;
4;6;
5;7;8;
6;12;
7;12;
8;4;
9;12;
10;12;
11;
